app/views/shared/_error_summary.html.erb

```diff
@@ -5,11 +5,13 @@
     </h2>
     <div class="govuk-error-summary__body">
       <ul class="govuk-list govuk-error-summary__list">
-        <% errors.each do |key| %>
+      <% errors.keys.each do |key| %>
```

Comment:
```<% errors.each do |key| %>
          <li data-turbolinks="false">
            <%= link_to errors.full_messages_for(key).first, "##{name}_#{key}" %>
          </li>
      <% end %>```

Reply:
that'd duplicate the errors... as there can be multiple errors for a single key